indico_toolkit/metrics/compare_ground_truth.py

```diff
@@ -19,7 +22,8 @@ def __init__(self, ground_truth: list, predictions: list):
 
     def set_all_label_metrics(self) -> None:
         self.all_label_metrics = {
-            label: self._get_base_metrics(label) for label in self.labels
+            label: self._get_base_metrics(label, self.span_type)
+            for label in self.labels
```

Comment:
Might want to add the span type here instead of class instantiation - 

If we wanted to check the metrics for different span types, the user currently would have to reinstantiate the class each time

Reply:
ooh, good call! will change now